src/languages/es.js

```diff
@@ -694,6 +694,16 @@ export default {
             setDefaultFailure: 'No se ha podido configurar el método de pago.',
         },
         addBankAccountFailure: 'Ocurrió un error inesperado al intentar añadir la cuenta bancaria. Inténtalo de nuevo.',
+        getPaidFaster: 'Recibe pagos más rápido',
+        addPaymentMethod: 'Agregue un método de pago para enviar y recibir pagos directamente en la aplicación.',
+        getPaidBackFaster: 'Recibe tu dinero más rápido',
+        secureAccessToYourMoney: 'Acceso seguro a su dinero',
+        receiveMoney: 'Recibe dinero en tu moneda local',
+        expensifyWallet: 'Expensify Wallet',
+        sendAndReceiveMoney: 'Envía y recibe dinero desde tu Expensify Wallet.',
```

Comment:
I think it should be 
`'Envía y recibe dinero desde tu billetera de Expensify.`